db/migrate/20211003230453_create_course_assessment_question_forum_post_responses.rb

```diff
@@ -27,16 +27,24 @@ def change
                      to_table: :course_discussion_posts
                    }
       t.string :post_text, null: false
-      t.boolean :is_post_updated, null: false
-      t.boolean :is_post_deleted, null: false
-      t.references :parent_post,
+      t.references :post_creator,
```

Comment:
I'm not sure if we want to track the creator and parent creator directly. Any rationale for doing so, besides making the table join less indirect? I think we can always access these by joining with the posts then with the creator through the posts.

Reply:
the creator info will be lost when the forum post gets deleted, thus storing the info here.